codebase2/codebase/U/Codebase/Reference.hs

```diff
@@ -22,7 +22,7 @@ type Id = Id' Hash
 data Reference' t h
   = ReferenceBuiltin t
   | ReferenceDerived (Id' h)
-  deriving (Eq, Ord, Show)
+  deriving (Eq, Ord, Show, Functor)
```

Comment:
Where are we using this, instead of `Bifunctor`?

Reply:
I guess I'd suggest using `bimap id` instead of `fmap` for `Reference'` if it's practical.